dev/ColorPicker/ColorSpectrum.cpp

```diff
@@ -328,32 +328,26 @@ void ColorSpectrum::RaiseColorChanged()
 {
     const winrt::Color newColor = Color();
 
-    if (m_oldColor.A != newColor.A ||
```

Comment:
What was the original purpose of this? Is it really appropriate to remove it?  In this case the color hasn't changed.. so raising the event doesn't really make sense, is that right?

Reply:
Problem is that when changing the "Value" is set 0, changing the other values can actually be a different color, it is just not in the RGB spectrum. In retrospective, maybe there is a way to detect this better instead of removing the check entirely?

Reply:
Could we add an additional check, "if both colors are black"?  That should be the only circumstance where the same RGB value can come from multiple HSV values.